qt/src/hocr/HOCRPdfExporter.cc

```diff
@@ -306,6 +306,17 @@ HOCRPdfExporter::HOCRPdfExporter(const HOCRDocument* hocrdocument, const HOCRPag
 	ui.lineEditPaperHeight->setValidator(new QDoubleValidator(0.0, 10000000.0, 1));
 	ui.lineEditPaperWidth->setValidator(new QDoubleValidator(0.0, 10000000.0, 1));
 
+	ui.comboBoxPdfVersion->addItem(_("1.0"), PoDoFo::EPdfVersion::ePdfVersion_1_0);
+	ui.comboBoxPdfVersion->addItem(_("1.1"), PoDoFo::EPdfVersion::ePdfVersion_1_1);
+	ui.comboBoxPdfVersion->addItem(_("1.2"), PoDoFo::EPdfVersion::ePdfVersion_1_2);
+	ui.comboBoxPdfVersion->addItem(_("1.3"), PoDoFo::EPdfVersion::ePdfVersion_1_3);
+	ui.comboBoxPdfVersion->addItem(_("1.4"), PoDoFo::EPdfVersion::ePdfVersion_1_4);
+	ui.comboBoxPdfVersion->addItem(_("1.5"), PoDoFo::EPdfVersion::ePdfVersion_1_5);
+	ui.comboBoxPdfVersion->addItem(_("1.6"), PoDoFo::EPdfVersion::ePdfVersion_1_6);
+	ui.comboBoxPdfVersion->addItem(_("1.7"), PoDoFo::EPdfVersion::ePdfVersion_1_7);
```

Comment:
I don't think it makes sense to translate these ;)

Reply:
Okay :-)